addons/calendar/static/tests/helpers/mock_server/models/mail_activity.js

```diff
@@ -15,16 +15,23 @@ patch(MockServer.prototype, {
      * @override
      */
     async _performRPC(route, args) {
-        if (args.model === 'mail.activity' && args.method === 'action_create_calendar_event') {
+        if (args.model === "mail.activity" && args.method === "action_create_calendar_event") {
             return {
-                type: 'ir.actions.act_window',
+                type: "ir.actions.act_window",
                 name: "Meetings",
-                res_model: 'calendar.event',
-                view_mode: 'calendar',
-                views: [[false, 'calendar']],
-                target: 'current',
+                res_model: "calendar.event",
+                view_mode: "calendar",
+                views: [[false, "calendar"]],
+                target: "current",
             };
         }
+        if (args.model === "mail.activity" && args.method === "unlink_w_meeting") {
+            const activities = this.getRecords("mail.activity", []);
```

Comment:
We should just use the correct domain instead of filtering every activity afterwards.

i.e. `[["id", "=", args.kwargs.id]]` (not sure about the exact path of the id.